src/consensus/mod.rs

```diff
@@ -35,10 +36,17 @@ pub struct FastPaxos<'a, C, B> {
     /// Channel the paxos instance will use to communicate with us about decision
     // paxos_rx: oneshot::Receiver<Vec<Endpoint>>,
     paxos: Option<Paxos<'a, C>>,
+    /// Latest configuration we accepted.
     config_id: ConfigId,
+    votes_received: HashSet<Endpoint>, // should be a bitset?
+    votes_per_proposal: HashMap<Vec<Endpoint>, AtomicUsize>,
```

Comment:
I think we can drop this `AtomicUsize` for a `usize`.

Reply:
Yeah probably, I figured a good first step would be to copy the logic, and then think about modifying it. Though I wonder why they chose to use an `AtomicUsize` for this when it doesn't look like the hasmap is being accessed from anywhere else.